iamproperty/iamproperty.github.io
# Ds 77/tabs

## Summary of Changes:
1. CSS
2. Vue component

## Rationale
Create a tabs component to group content together and only display parts of it at once.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/tabs

## Ticket(s)
DS-77

## Notes
